docs/grpc/docs.md

```
@@ -2437,6 +2438,28 @@ The JSON representation for `Value` is a JSON value.
 
 
 
+<a name="qdrant-AcornSearchParams"></a>
+
+### AcornSearchParams
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| enable | [bool](#bool) | optional | If true, then ACORN may be used for the HNSW search based on filters selectivity.
```

markdownlint-cli2 (0.18.1)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)